src/OpenLoco/Audio/Audio.cpp

```diff
@@ -613,7 +613,7 @@ namespace OpenLoco::Audio
                     return;
                 }
 
-                volume += calculateVolumeFromViewport(id, { loc.x, loc.y }, *viewport);
+                volume += calculateVolumeFromViewport(id, { loc.x, loc.y, 0 }, *viewport);
```

Comment:
This did previously always pass ```z = 0``` because the implementation allowed implicit construction of z which is always zero. The new implementation does not allow this and it should never allow it, things like this can happen.